lib/utils/download-template-from-repo.js

```diff
@@ -291,6 +291,7 @@ async function downloadTemplateFromRepo(inputUrl, templateName, downloadPath) {
 
     if (isPlainGitURL(inputUrl)) {
       return spawn('git', ['clone', inputUrl, downloadServicePath]).then(() => {
+        fse.removeSync(path.join(dirName, '/.git'));
```

Comment:
Let's in first PR refactor `downloadTemplateFromRepo` function into async/await, and then introduce this change, but using _async_ fs interface

Reply:
will do.  Thanks for note.